postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -51,25 +52,28 @@
 EXPLAIN_VALIDATION_QUERY = "SELECT * FROM pg_stat_activity"
 
 
-class DBExplainSetupState(Enum):
+class DBExplainError(Enum):
     """
-    Denotes the various states the database can be in with respect to enabling the agent to collect execution plans.
+    Denotes the various reasons a query may not have an explain statement.
     """
 
-    # agent is able to collect execution plans using the configured explain function
-    ok = 1
+    # database error i.e connection error
+    database_error = 'database_error'
 
-    # failed to connect to the database
-    failed_connect = 2
+    # statements not in VALID_EXPLAIN_STATEMENTS are not valid
+    invalid_statement = 'invalid_statement'
```

Comment:
Unused

Reply:
It's still there and unused. Did you intend to use it or to remove it?

Reply:
Ah yes, my mistake, I intended to remove it in the last commit. I may have undid the changes without noticing 🤦‍♂️.